attribute-service-api/src/main/proto/org/hypertrace/core/attribute/service/v1/attribute_metadata.proto

```diff
@@ -54,9 +54,10 @@ enum AttributeType {
 
 enum AttributeSource {
```

Comment:
i'd vote to convert this into a string before we make further changes. It will be totally worth.